cfr.yaml

```diff
@@ -1335,3 +1335,68 @@
     id: contentBlocking
     params:
     - 2048
+- id: IMPORT_PASSWORD
```

Comment:
nit: Can we make this a bit more specific? Experiment, Fx Version, `CFR_`.

Reply:
sure thing.